### ISSUE TITLE
Where will we put lonoa on production server

### ISSUE THREAD

**Issue by matthew-schultz** [erdl/lonoa#100]
Useful to know in case Eileen has to move the code to a different server

- check online where other people put similar scripts
    - check best practices
- compare to scrape-util's location
   - seems to be in /usr/local/lib/scrape-util
   - configuration files
      - /var/local/scrape-util/projects

**Comment by matthew-schultz** [comment]
@carlosparadis 

After some research, I was thinking we could use a directory like this to store the lonoa projects: 
```/usr/local/bin/lonoa-projects/```

Why ```/usr/local/bin/```
- /usr/local is a shared directory for the whole system
  > These days, '/usr/local' is widely regarded as a good place in which to keep self-compiled or third-party programs 
https://www.tldp.org/LDP/Linux-Filesystem-Hierarchy/html/usr.html
- /bin directories tend to be used for binaries or programs that you run
   > /usr/local/bin is for normal user programs not managed by the distribution package manager, e.g. locally compiled packages. 
https://unix.stackexchange.com/questions/8656/usr-bin-vs-usr-local-bin-on-linux

Proposed usage when creating new projects
---
- Create a new directory named after the project in ```/usr/local/bin/lonoa-projects/``` and then clone lonoa into that created directory.

**Comment by matthew-schultz** [comment]
As of 8/7/19 we set up the lonoa_frogs project in ```/home/lonoa```

This will:
- help ensure that the lonoa user has file permissions to run properly (writing to logs, etc)
- keeps lonoa project directories in one place that should be fairly easy to remember (other directories like /usr/local/lib might be more difficult to remember)
   - we decided not to use /var/log/ to store logs. While using /var/log might be more in line with the Linux common practices, storing the logs inside the repos should make them easier for users to find.

Also closing this issue, since the location has been decided.
